### ISSUE TITLE
FIX: Fileinput Radiobutton - focus outline missing

### ISSUE THREAD

**Issue by NovaCat35** [ITS-HCD/nysds#1731]
## Summary
The blue focus outline when you hit submit button within a `<form>` is missing from `nys-fileinput` and `nys-radiobutton` when `required`
